apps/frontend/src/components/global/upload_tabs/FileReader.vue

```diff
@@ -27,6 +27,7 @@
             <li>Static Analysis Results Interchange Format (SARIF)</li>
             <li>Scoutsuite</li>
             <li>Snyk</li>
+            <li>Tenable Export (nessus)</li>
```

Comment:
Change "Tenable Export (nessus)" to "Tenable (Nessus)". Export is confusing because the user might confuse that with the different Heimdall export options and we don't need to get into all of that at this time.

Reply:
Should I use Tenable.SC (Nessus)